tripl/nestly.py

```diff
@@ -16,7 +14,10 @@
 
 import bio
 
-#import tripl
+# pp  # lint
```

Comment:
This line was active before, but I could not figure out what the expression `pp` is supposed to do. PyCharm suggested it would have no effect at all.

Reply:
The reason I did that is that I get warnings in Vim when I import a module and then don't use it. So here I was just "using it" as a no-op so that the vim linter doesn't complain any more.